README.md

```diff
@@ -127,39 +132,39 @@ If this is the case, shut down that MongoDB and run the `docker run` command aga
 - Editing a Raincatcher module locally and see it reflected in the Demo solution immediately.
 
 ### Running The Demo Solution Locally
+
 1. Install *tmux*
   * On linux:
-```
-apt-get install tmux
-```
-  * On Mac, this may work:
-> https://gist.github.com/simme/1297707
+    ```
+    apt-get install tmux
+    ```
+  * On Mac, this may work: https://gist.github.com/simme/1297707.
 
 2. Install tmuxinator (ruby required)
-```
-gem install tmuxinator
-```
-3. Start the WFM node processes (mongo will be started automatically if you installed it locally, else the Docker container should already be running):
-```
-wfm start
-```
+  ```
+  gem install tmuxinator
+  ```
+
+3. Start the Raincatcher node processes (mongo will be started automatically if you installed it locally, else the Docker container should already be running):
```

Comment:
I think we're missing a step to install `grunt`—does anyone know if that was left out for a reason?